src/ui/overview.rs

```diff
@@ -30,27 +30,27 @@ pub fn draw_overview<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rect) {
   }
 }
 
-fn draw_status_block<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rect) {
+fn draw_status_block<B: Backend>(f: &mut Frame<'_, B>, app: &mut App, area: Rect) {
   let chunks = horizontal_chunks(
     vec![
-      Constraint::Length(30),
+      Constraint::Length(35),
       Constraint::Min(10),
-      Constraint::Length(40),
       Constraint::Length(30),
+      Constraint::Length(32),
     ],
     area,
   );
 
-  draw_cli_version_block(f, app, chunks[0]);
+  draw_namespaces_block(f, app, chunks[0]);
```

Comment:
Not related to this PR, there is a stack overflow error when key `n` is pressed twice, it doesn't happen until release 0.2.6, so I need investigate what changed